### PR TITLE
format the repo and add csv support in create options

### PR DESCRIPTION
**Pull request by hans-lizihan**
@adamliuxy @suddi 

1. update style
2. update an option in the create method to support create label from csv.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,30 +104,21 @@ postmen.get('/rates/put-your-label-id-here', function (err, result) {
 Initiate Postmen SDK object.
 In order to get API key and choose a region refer to the [documentation](https://docs.postmen.com/overview.html).
 
-| Argument                       | Required                               | Type    | Default   | Description                                       |
```
Comment (suddi) [inline README.md:107]: Why remove all of this?

File: lib/payload.js [hunk 2]
```diff
@@ -70,6 +70,10 @@ class Payload {
 			postmen.tmp_api_key = null;
 		}
 
+		if (config.csv) {
```
Comment (suddi) [inline lib/payload.js]: Can we keep this generic, rather than configuring specifically for `csv`?

Comment (suddi) [reply lib/payload.js]: We just pass additional headers, and handle it in the code.

**Review by suddi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hans-lizihan) [reply README.md:107]: whoops

File: lib/payload.js (hunk 2 above)
Comment (hans-lizihan) [reply lib/payload.js]: no, it seems that the SDK doesn't provide such functionality to customize the headers.

I could do so, but allowing a customised header may involves more changes to this SDK.

File: README.md (hunk 1 above)
Comment (hans-lizihan) [reply README.md:107]: fixed.

**Review by suddi — APPROVED** [review]

**Pull request merged**